.github/workflows/ci.yml

```diff
@@ -96,7 +106,7 @@ jobs:
     - name: cmake
       run: |
         mkdir -p out
-        cmake -S . -B out -G Ninja -DCMAKE_C_COMPILER=gcc-7 -DCMAKE_CXX_COMPILER=g++-7
+        cmake -S . -B out -G Ninja -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++
```

Comment:
why change the compiler here? (why not remove a mention of the compiler as in the change above?)

Reply:
I decided to make the default "linux" build use the default compiler and have a special "clang" build on the side.  Rather than having "clang" be the default build.